Makefile

```diff
@@ -704,3 +715,10 @@ static-analysis: dqlite-install-if-missing
 	@cd tests && CGO_ENABLED=1 \
 		CGO_LDFLAGS_ALLOW="(-Wl,-wrap,pthread_create)|(-Wl,-z,now)" \
 		./main.sh static_analysis ${STATIC_ANALYSIS_JOB}
+
+.PHONY: rev-build-number
+rev-build-number:
+## rev-build-number: increase the builder number
+	@mkdir -p $(BUILD_DIR)
+	@printf $$(($(JUJU_BUILD_NUMBER)+1)) > $(BUILD_DIR)/build.number
```

Comment:
Would it be possible to make the location of this file configurable? I use Git worktrees so I have multiple build directories (one for each branch), it would be nice to have a shared build counter for all branches though